azkaban-common/src/test/java/azkaban/executor/InteractiveTestJob.java

```diff
@@ -24,7 +26,7 @@
 
 public class InteractiveTestJob extends AbstractProcessJob {
 
-  private static final ConcurrentHashMap<String, InteractiveTestJob> testJobs =
+  public static final ConcurrentHashMap<String, InteractiveTestJob> testJobs =
```

Comment:
Now I realized why it is `static`: because this way we can affect "job type" behaviour in the tests. Basically to tell it to succeed or fail, when it's waiting for that kind of instruction (there are some test cases that wouldn't be possible otherwise). Other option would be to somehow expose the private field `Job job` in `JobRunner`, so that the test could get it, cast to `InteractiveTestJob` and call `succeedJob` or `failJob`. I'm not going to make any change to that.